contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -7527,10 +7532,12 @@ exec_eval_simple_expr(PLtsql_execstate *estate,
 
 	/*
 	 * Finally we can call the executor to evaluate the expression
+	 *  if not in explain only mode
 	 */
-	*result = ExecEvalExpr(expr->expr_simple_state,
-						   econtext,
-						   isNull);
+	if (!pltsql_explain_only)
```

Comment:
When do we need this?

Reply:
this is not needed any longer.  Its an artifact of trying to get explain to work with function calls